### ISSUE TITLE
create UI ingestion encounter "cx_Oracle.DatabaseError"

### ISSUE THREAD

**Issue by Amanda-Chgar** [datahub-project/datahub#4514]
**Describe the bug**
Using UI ingestion oracle databases, report error:

 'DatabaseError: (cx_Oracle.DatabaseError) DPI-1047: Cannot locate a 64-bit Oracle Client library: "libclntsh.so: cannot open shared '

**To Reproduce**
1. Go to 'top tool bar, ingestion, create oracel ingestion yml'
2. Click on 'execute'
3. Scroll down to 'detail'

**Expected behavior**
how to fix this problem exactly?
<img width="960" alt="1648524323402_8DE015A0-5E13-48bd-B653-08C9294EBB69" src="https://user-images.githubusercontent.com/60022065/160527037-df674c64-e8a5-427e-8d4a-a114502797bd.png">

**Comment by maggiehays** [comment]
This looks realted to https://github.com/datahub-project/datahub/issues/4486

**Comment by jjoyce0510** [comment]
Yes it is. I'm going to close this issue due to inactivity, and we can continue to track it there.
